tenant_schemas/middleware.py

```diff
@@ -75,22 +73,28 @@ class SuspiciousTenantMiddleware(TenantMiddleware):
     TENANT_NOT_FOUND_EXCEPTION = DisallowedHost
 
 
-class DefaultSchemaTenantMiddleware(TenantMiddleware):
+class DefaultTenantMiddleware(SuspiciousTenantMiddleware):
```

Comment:
Why use SuspiciousTenantMiddleware? It seems to me when the DefaultTenant can't be found it would be better to raise a 404 than a 400.